onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -821,6 +821,11 @@ def _where_input_wrangler(
         matcher=lambda sample: not (sample.args[0][0].dtype == torch.bool),
         reason="this Aten overload only support tensor(bool) as args",
     ),
+    skip(
+        "matmul",
+        matcher=lambda sample: torch.numel(sample.input) == 0,
```

Comment:
what is relationship of `sample.input` and `sample.args` btw?

Reply:
input is the first input. args[...] starts from the second